src/components/TodoItem/TodoItem.tsx

```diff
@@ -0,0 +1,71 @@
+import classNames from 'classnames';
+import {
+  useAppDispatch,
+  useAppSelector,
+} from '../../app/hooks';
+import { actions as todoActions } from '../../features/currentTodo';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  todo: Todo,
+};
+
+export const TodoItem: React.FC<Props> = ({ todo }) => {
+  const dispatch = useAppDispatch();
+
+  const currentTodo = useAppSelector(state => state.currentTodo);
+
+  const handleSelectTodo = (item: Todo) => {
+    dispatch(todoActions.setTodo(item));
+  };
+
+  return (
+    <tr
+      className={classNames(
+        currentTodo?.id === todo.id
+          ? 'has-background-info-light'
+          : '',
+      )}
```

Comment:
Please fix all cases with classes

```suggestion
      className={classNames({ 'has-background-info-light':  currentTodo?.id === todo.id })}
```

Reply:
Done ✅